src/tutorials/0005/01.md

```diff
@@ -19,7 +19,7 @@ The Files API, on the other hand, is custom-built for a more specific use case.
 
 If you've read our [Mutable File System tutorial](https://proto.school/#/mutable-file-system), you may be thinking, "I've already learned how to work with files on IPFS. How will this be any different?"
 
-The Mutable File System (MFS) provides an API designed to replicate familiar file system operations such as `mkdir`, `ls`, `cp`, and others, mimicking the way you organize files and directories on a computer. However, the way that content is addressed in IPFS makes it an immutable file system. The address to a file or directory depends on its contents, so any change to a file or directory will result in an entirely new address. The MFS Files API works on a familiar-looking file system with regular paths — like `/some/stuff` — in the local IPFS node, which hides the complexity of immutable content addressing. You can recognize MFS API methods in js-ipfs because they take the following format:
+The Mutable File System (MFS) provides an API designed to replicate familiar file system operations such as `mkdir`, `ls`, `cp`, and others, mimicking the way you organize files and directories on a computer. However, the way that content is addressed in IPFS makes it an immutable file system. The address to a file or directory depends on its contents, so any change to a file or directory will result in an entirely new address. The MFS Files API works on a familiar-looking file system with regular paths — like `/some/stuff` — in the local IPFS node, which hides the complexity of immutable content addressing. You can recognize MFS API methods in js-ipfs because they take the following format: `ipfs.files.someMethod()`
```

Comment:
Nice catch @dominguesgm. This was actually my bad; I meant to remove that whole sentence because I shared this info further down. Just included the fix in another commit.